packages/opentelemetry-instrumentation-redis/opentelemetry/instrumentation/redis/__init__.py

```diff
@@ -0,0 +1,74 @@
+"""OpenTelemetry Redis instrumentation"""
+
+from opentelemetry.instrumentation.redis.config import Config
+from opentelemetry.instrumentation.redis.version import __version__
+from opentelemetry.instrumentation.redis.wrapper import _wrap
+from opentelemetry.instrumentation.instrumentor import BaseInstrumentor
+from opentelemetry.instrumentation.utils import unwrap
+from opentelemetry.trace import get_tracer
+
+import logging
+import redis
+import redis.commands.search
+from typing import Collection
+from wrapt import wrap_function_wrapper
+
+
+logger = logging.getLogger(__name__)
+
+_instruments = ("redis >= 4.6.0",)
+
+# The ping and create_index methods are not shown on the Web UI due to filtering
+WRAPPED_METHODS = [
```

Comment:
should we also trace 
for querying:
- ft.aggregate
for ingestion:
- HSET 
- JSON.SET 
?

Reply:
Marcin added.